projects/cdk/utils/dom/get-viewport-height.ts

```diff
@@ -0,0 +1,6 @@
+export function getViewportHeightBy(windowRef: Window): number {
```

Comment:
`tui`-prefix ?

Reply:
Also `By` is unnecessary and let's destruct :)

Reply:
fixed